extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -0,0 +1,143 @@
+
+# KHR_lights_environment
+
+## Contributors
+
+* Norbert Nopper, <mailto:nopper@ux3d.io>
+* Rickard Sahlin, <mailto:rickard.sahlin@ikea.com>
+* Gary Hsu, Microsoft, <mailto:garyhsu@microsoft.com>
+* Mike Bond, Adobe, <mailto:mbond@adobe.com>
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+This extension may use KHR_texture_basisu, KHR_texture_float_bc6h or KHR_texture_float_rgb9e5  
```

Comment:
`KHR_texture_basisu` is strictly limited to 8-bit data. How is it used with HDR IBL values?

Reply:
This extension shall not mandate that an HDR IBL _must_ be used.  
If wanted an LDR image may be used.  
I'm also thinking that we may need some other mechanism to provide uncompressed 8 bit data for environment maps that does not need HDR but want to provide a lossless image (like PNG).

Reply:
I think that using LDR is not recommended.  Do we need this flexibility?  I feel it makes the extension have larger surface area? Could we be prescriptive to best practices only?

Reply:
From a technical perspective I understand - but I'm all for a specification that allows for flexibility in use.
If wanted that should include use of LDR images, this could for instance be to save transmission size or GPU memory.

My hope is to have a specification that sort of seamlessly accepts a cubemap containing _any_ supported image format in viewer/engine. 
Hopefully the inclusion of LDR (existing JPG/PNG images) textures will not make the extension bigger or harder to implement.

Reply:
Extension is now using KTX 2 and supports formats with higher range (HDR)
I think this conversation is resolved.